src/main/java/de/uhd/ifi/se/decision/management/jira/filtering/FilterSettings.java

```diff
@@ -538,4 +559,75 @@ public String toString() {
 		}
 		return filterSettingsAsJson;
 	}
+
+	public long getContext() {
+		return context;
+	}
+
+	@JsonProperty("context")
+	public void setContext(long context) {
+		this.context = context;
+	}
+
+	public String getDisplayType() {
+		return displayType;
+	}
+
+	@JsonProperty("displayType")
+	public void setDisplayType(String displayType) {
+		this.displayType = displayType;
+	}
+
+	public Map<String, Float> getLinkImpact() {
+		return linkImpact;
+	}
+
+	@JsonProperty("linkImpact")
+	public void setLinkImpact(Map<String, Float> linkImpact) {
+		this.linkImpact = linkImpact;
+	}
+
+	public double getDecayValue() {
```

Comment:
Instead of passing these single parameters you could think of passing a CiaSetting object. Otherwise I think shot comments would be good for these parameters.